ledger-tool/src/main.rs

```diff
@@ -963,6 +963,9 @@ fn main() {
         .validator(is_slot)
         .takes_value(true)
         .help("Halt processing at the given slot");
+    let no_os_memory_stats_reporting_arg = Arg::with_name("no_os_memory_stats_reporting")
```

Comment:
This is a different change for adding no_os_memory_stats_reporting cli args to ledger-tool. It is not relevant to the memory map.